nav2_system_tests/worlds/turtlebot3_ros2_demo.world

```diff
@@ -153,6 +153,8 @@
           <plugin name="my_imu_plugin" filename="libgazebo_ros_imu_sensor.so">
             <ros>
               <!-- <namespace>/tb3</namespace> -->
+              <argument>--ros-args</argument>
+              <argument>-r</argument>
```

Comment:
I believe the `-r` flag should be placed before every remapping, in which case I'd place the flag inside the argument description, for example:
```cpp
<argument>-r ~/out:=imu</argument>
<argument>-r ~/from_remap1:=to_remap1</argument>
<argument>-r ~/from_remap2:=to_remap2</argument>
```

Reply:
I tried that and it didn't work for me